internal/api/v1beta1/org.go

```diff
@@ -259,9 +261,67 @@ func (h Handler) ListOrganizationUsers(ctx context.Context, request *frontierv1b
 		}
 	}
 
-	users, err := h.userService.ListByOrg(ctx, orgResp.ID, request.GetPermissionFilter())
-	if err != nil {
-		return nil, err
+	var users []user.User
+	var rolePairPBs []*frontierv1beta1.ListOrganizationUsersResponse_RolePair
+
+	if len(request.GetRoleFilters()) > 0 {
+		// convert role names to ids if needed
```

Comment:
Can we make this compatible with `with_roles` field as well?

Reply:
Because as an API contract, we should not have completely different behaviours depending on the filters being passed.

Reply:
it doesn't make sense to return roles when you are querying by roles.

Reply:
Added validation to fail if both are passed.